src/lib/mdns/minimal/BUILD.gn

```diff
@@ -15,6 +15,8 @@
 import("//build_overrides/chip.gni")
 
 static_library("minimal") {
+  output_name = "libMdns"
```

Comment:
Can you not set `chip_monolithic_tests = true` to avoid needing to change this?

Reply:
It is actually set to true when tests are built: https://github.com/project-chip/connectedhomeip/blob/1442adb8af300f05434c99ea17697bfb889c73d3/config/mbed/CMakeLists.txt#L211 . We can remove this I suppose.

Reply:
Fixed - changes revert